RELEASE.md

```diff
@@ -0,0 +1,35 @@
+# Lodestar Release Guidelines
+
+![lodestar-release](https://user-images.githubusercontent.com/58080811/148576989-0e6924f5-8b5a-48c8-baa5-8a833cc364b0.png)
```

Comment:
This image must be persisted in a more stable place, like committed to the code at `docs/images`

Reply:
Uploaded to docs/images and linked in new RELEASE.md commit 3cd8fcf